### DISCUSSION TITLE
Multi-line lists

### DISCUSSION THREAD

**Discussion by bcgoss** [kdl-org/kdl#142] [Q&A] [answered]
I love the look and feel of this so far. One use case I imagined was a list that got very long, say a list of admins with 20 emails. It would run off the right side of the page.

`admins "bcgoss@gmail.com" "foo@example.com" "bar@example.com" ... `

With multi line strings I could just make the whole list a big string but the concern also applies to a long list of numbers. Is there a way to break this long list into multiple lines? If so can we clarify the readme to show it? If not what would it look like? As a discussion starter I propose square brackets

```
admins [
  "bcgoss@gmail.com"
  "foo@exmaple.com"
  "bar@example.com"
  ...
]
```

Apologies if this was discussed elsewhere I tried to read everything that seemed relevant but there's a lot conversations happening and I might have missed it.

**Comment by zkat** [comment, marked as answer]
Yes! You can use `linespace`s for this:

```kdl
admins \
  "bcgoss@gmail.com" \
  "foo@exmaple.com" \
  "bar@example.com"
```

Though in general, I recommend switching to children if you expect something to become unbounded like this. If you want to be clever, you can even use the _node names_ as string values:

```kdl
admins {
  "bcgoss@gmail.com"
  "foo@exmaple.com"
  "bar@example.com"
  ...
}
```

But use whatever you feel fits your needs better! KDL is flexible~
